build.sbt

```diff
@@ -213,7 +213,7 @@ lazy val V = new {
   def deprecatedScala2Versions =
     Seq(scala211, "2.12.8", "2.12.9", "2.13.0", "2.13.1")
   def nonDeprecatedScala2Versions =
-    Seq(scala213, scala212, "2.12.11", "2.12.10", "2.13.2", "2.13.3")
+    Seq(scala213, scala212, "2.12.13", "2.12.11", "2.12.10", "2.13.2", "2.13.3")
```

Comment:
I decided to add support for 2.12.13, but keep Metals on 2.12.12 until we can just calmly update. I should have done it in the beginning, but it seemed I would be able to just do one more thing and they just kept poping up.

Reply:
No worries at all. We can easily just do it after.